pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -864,16 +1365,47 @@ func (pl *DynamicResources) bindClaim(ctx context.Context, state *stateData, ind
 					logger.V(5).Info("Claim not stored in assume cache", "err", finalErr)
 				}
 			}
-			pl.draManager.ResourceClaims().RemoveClaimPendingAllocation(claim.UID)
+			for _, claimUID := range claimUIDs {
+				pl.draManager.ResourceClaims().RemoveClaimPendingAllocation(claimUID)
+			}
 		}
 	}()
 
-	logger.V(5).Info("preparing claim status update", "claim", klog.KObj(state.claims[index]), "allocation", klog.Format(allocation))
+	// Create the special claim for extended resource backed by DRA
+	if isExtendedResourceClaim && claim.Name == specialClaimInMemName {
+		logger.V(5).Info("preparing to create claim for extended resources", "pod", klog.KObj(pod), "node", klog.ObjectRef{Name: nodeName}, "resourceclaim", klog.Format(claim))
```

Comment:
Just pass a nodeName:
```suggestion
		logger.V(5).Info("preparing to create claim for extended resources", "pod", klog.KObj(pod), "node", nodeName, "resourceclaim", klog.Format(claim))
```

Reply:
Similarly in other logs below